src/v2/guide/list.md

```diff
@@ -329,29 +329,29 @@ new Vue({
 </script>
 {% endraw %}
 
-## key
+## key (clave)
 
-When Vue.js is updating a list of elements rendered with `v-for`, it by default uses an "in-place patch" strategy. If the order of the data items has changed, instead of moving the DOM elements to match the order of the items, Vue will simply patch each element in-place and make sure it reflects what should be rendered at that particular index. This is similar to the behavior of `track-by="$index"` in Vue 1.x.
+Cuando Vue.js está actualizando una lista de elementos renderizados con `v-for`, por defecto usa una estrategia "arreglar en-lugar". Si el orden de los elementos de datos cambia, en vez de mover los elementos DOM para ajustar el orden de los elementos, Vue sencillamente arregla cada elemento en su lugar y se asegura que refleje lo que debe ser renderizado en ése índice particular. Ésto es similar al comportamiento de `track-by="$index"` en Vue 1.x.
```

Comment:
Dejaría `"in-place patch"` sin traducir.

Reply:
hecho